pkg/scaffold/operator.go

```diff
@@ -55,9 +55,6 @@ spec:
         - name: {{.ProjectName}}
           # Replace this with the built image name
           image: REPLACE_IMAGE
```

Comment:
So I removed the below as it is not strictly necessary. It would be difficult to keep this up to date with the port the user passes to controller-runtime.

Reply:
This would correspond to the port in `metricsAddress` in `main.go`, right? I'd suggest that we leave the port definition in with a comment that says it should match the port defined in `metricsAddress` in `cmd/manager/main.go`. That way it's on by default and users don't have to know to insert these lines to get it to work.

Reply:
Yes, the thing is this is primarily informational purpose https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#container-v1-core and I am afraid of confusing the user if it's out of date. Which is why I removed it in the first place. It would be an additional step that is not strictly necessary to do. But if we decide to leave it in, should we leave in the default value `8080` or `REPLACE_PORT` so it fails when creating the deployment?

Reply:
TIL about the port definition being informational! :+1: 

In that case, I agree with leaving it out.